source/extensions/filters/http/rbac/rbac_filter.h

```diff
@@ -22,7 +22,7 @@ class RoleBasedAccessControlRouteSpecificFilterConfig : public Router::RouteSpec
   RoleBasedAccessControlRouteSpecificFilterConfig(
       const envoy::extensions::filters::http::rbac::v3::RBACPerRoute& per_route_config);
 
-  const Filters::Common::RBAC::RoleBasedAccessControlEngineImpl*
+  Filters::Common::RBAC::RoleBasedAccessControlEngineImpl*
```

Comment:
returning non-const seems wrong here. Why is this needed? Same below.

Reply:
The handleAction method called from the engine is not const. So, I don't think the object pointer can be const.

Reply:
I think we need to make it const. It doesn't make sense for the engine to modify itself. Why does it need to be non-const?

Reply:
My mistake, actually handleAction doesn't need to be non-const, I'll change that.